telecine/src/main/java/com/jakewharton/telecine/TelecineActivity.java

```diff
@@ -5,18 +5,21 @@
 import android.content.Intent;
 import android.content.res.Resources;
 import android.graphics.Bitmap;
-import android.graphics.drawable.BitmapDrawable;
+import android.graphics.BitmapFactory;
 import android.os.Bundle;
 import android.widget.Spinner;
 import android.widget.Switch;
+
+import com.google.android.gms.analytics.HitBuilders;
+
+import javax.inject.Inject;
```

Comment:
In general, be careful not to do reformatting. This project uses the 'SquareAndroid' code style from http://github.com/square/java-code-styles/.